api/app/main.py

```diff
@@ -132,9 +132,10 @@ def get_noon_forecasts(request: schemas.StationCodeList, _: bool = Depends(authe
     set of weather stations. """
     try:
         LOGGER.info('/noon_forecasts/')
-        start_date = datetime.datetime.now(tz=datetime.timezone.utc)
-        end_date = start_date + datetime.timedelta(days=5)
-        return fetch_noon_forecasts(request.stations, start_date, end_date)
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
Once my test PR is merged, please switch to using app.time_utils.get_uct_now - it makes mocking for unit tests real easy if we mock out .now functionality in the same place.